crates/qt-build-utils/src/parse_cflags.rs

```diff
@@ -103,7 +103,7 @@ fn split_flags(link_args: &[u8]) -> Vec<String> {
     words
 }
 
-pub(crate) fn parse_libs_cflags(name: &str, link_args: &[u8]) {
+pub(crate) fn parse_libs_cflags(_builder: &mut cc::Build, name: &str, link_args: &[u8]) {
```

Comment:
Move `builder` to be the last argument and add a TODO to remove it (see other review comment).

Reply:
I added the TODO comment at the same place where builder is used.

Also _builder can't drop its leading _ because clippy would complain without the feature flag turned on.